src/applications/vaos/services/appointment/index.js

```diff
@@ -263,16 +264,25 @@ export function hasValidCovidPhoneNumber(facility) {
 }
 
 /**
- * Checks to see if a past appointment has a valid status
+ * Checks to see if an appointment should be shown in the past appointment
+ * list
  *
  * @param {Appointment} appt A FHIR appointment resource
  * @returns {boolean} Whether or not the appt should be shown
  */
 export function isValidPastAppointment(appt) {
   return (
-    appt.vaos.appointmentType !== APPOINTMENT_TYPES.vaAppointment ||
-    (!PAST_APPOINTMENTS_HIDDEN_SET.has(appt.description) &&
-      !appt.vaos.isExpressCare)
```

Comment:
EC would never be a confirmed type, so it's not necessary now in the updated check